packages/flutter/pubspec.yaml

```diff
@@ -23,13 +23,12 @@ dev_dependencies:
   flutter_goldens:
     sdk: flutter
   mockito: 4.1.1
-  quiver: 2.1.3
+  fake_async: 1.1.0-nullsafety
```

Comment:
Just to double-check: No other pubspec file changed as a result of running `flutter update-packages --force-upgrade`?

Reply:
yes, since this was a dev_dependency it doesn't affect the version solving of any other package